core-bundle/contao/library/Contao/Combiner.php

```diff
@@ -122,17 +120,9 @@ public function add($strFile, $strVersion=null, $strMedia='all')
 		}
 
 		// Check the source file
-		if (!file_exists($this->strRootDir . '/' . $strFile))
+		if (!file_exists($this->strWebDir . '/' . $strFile))
 		{
-			// Handle public bundle resources in the contao.web_dir folder
-			if (file_exists($this->strRootDir . '/' . $this->strWebDir . '/' . $strFile))
-			{
-				$strFile = $this->strWebDir . '/' . $strFile;
-			}
-			else
-			{
-				return;
-			}
+			return;
 		}
```

Comment:
This would break BC for adding files that are not symlinked to the web dir I think.
For example SCSS files from a non-public `files/` folder. _(Did not test that manually yet)_